app/src/main/java/com/android/sample/ui/composables/CameraFeedback.kt

```diff
@@ -103,6 +104,7 @@ class CameraFeedBack {
                       }
                 })
           }
+      DisposableEffect(Unit) { onDispose { cameraViewModel.cameraController.value.unbind() } }
```

Comment:
This is not quite what i thought of. In my opinion the binding and unbinding part should be in the same scope.
```
DisposableEffect(lifecycleOwner) {
    val controller = cameraViewModel.cameraController.value
    controller.bindToLifecycle(lifecycleOwner)
    onDispose { controller.unbind() }
}
```
In general i like to think of the disposable effect the way I think of freeing the pointers in c :)

The general syntax is the following:
```
DisposableEffect(Unit) {
    // Code to execute when the composable is first placed in the composition.
    
    onDispose {
        // Code to run when the composable leaves the composition.
    }
}

```

the on dispose method being good for cleaning the resources that you used before.